### PR TITLE
Finished Signup screen UI and styling

### PR DESCRIPTION
**Pull request by OphirSheriff**

### CODE REVIEW

File: lineQueueClient/src/components/Header/Header.js [hunk 1]
```diff
@@ -1,49 +1,46 @@
...
  */
 const Header = () => {
   let headerStyle = StyleSheet.create({
+    headerContainer: {
+      flex:1,
+      backgroundColor: '#151515',
+      maxHeight: 220,
+      justifyContent: 'flex-start',
+      alignItems:'flex-end',
+      paddingRight: 35,
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Header/Header.js]: Remove static positioning

File: lineQueueClient/src/components/Header/Header.js [hunk 2]
```diff
@@ -1,49 +1,46 @@
...
+import {Text, View, StyleSheet, Image,} from 'react-native';
 
 /**
  * Renders the app header
  */
 const Header = () => {
   let headerStyle = StyleSheet.create({
+    headerContainer: {
+      flex:1,
+      backgroundColor: '#151515',
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Header/Header.js]: `#151515` should be global variable

File: lineQueueClient/src/components/Header/Header.js [hunk 3]
```diff
@@ -1,49 +1,46 @@
...
+      maxHeight: 220,
+      justifyContent: 'flex-start',
+      alignItems:'flex-end',
+      paddingRight: 35,
+      paddingTop:25
+    },
     logo: {
       width: 103,
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Header/Header.js]: Again, more static positioning

File: lineQueueClient/src/components/Header/Header.js [hunk 4]
```diff
@@ -1,49 +1,46 @@
...
+      paddingTop:20,
       fontSize: 40,
+      lineHeight: 35,
       textAlign: 'right',
       color: 'white',
-      paddingRight: 30,
 			fontFamily: 'Alef',
-			fontWeight: 'bold'
-    }
+			fontWeight: 'bold'    }
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Header/Header.js]: bad code style, provide uniform indentation

File: lineQueueClient/src/components/Header/Header.js [hunk 5]
```diff
@@ -1,49 +1,46 @@
...
-      }}/>
-		<Image style={headerStyle.logo} source={require('../../images/BezalelLogo.png')}></Image>
-      <View style={{
-        height: 20
-      }}/>
+    <View style={headerStyle.headerContainer}>
+      <View />
+      <Image style={headerStyle.logo} source={require('../../images/BezalelLogo.png')}></Image>
+
+      <Text style={headerStyle.title}>
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Header/Header.js]: Remove unnecessary empty lines

File: lineQueueClient/src/components/Signup/Signup.js [hunk 6]
```diff
@@ -1,8 +1,9 @@
 import React, {Component} from 'react'
-import {Text, View, TextInput, Button} from 'react-native';
+import {Text, View, TextInput, TouchableHighlight} from 'react-native';
 import Header from '../Header/Header'
+import globals from '../../styles/globals.style'
 import signupStyle from './Signup.style'
-import { Actions } from 'react-native-router-flux'
+import {Actions} from 'react-native-router-flux'
 
 export default class Signup extends Component {
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Signup/Signup.js]: Add documentation
short explanation about component, similar to `Header` component

File: lineQueueClient/src/components/Signup/Signup.js [hunk 7]
```diff
@@ -14,23 +15,29 @@ export default class Signup extends Component {
 
   render() {
     return (
-      <View style= color="#000000">
+      <View style={globals.sceneContainer}>
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Signup/Signup.js]: Remove unnecessary empty lines

File: lineQueueClient/src/components/Signup/Signup.style.js [hunk 8]
```diff
@@ -1,33 +1,58 @@
...
   },
   box: {
-    height: 58,
-    width: 264,
-    borderColor: '#404040',
+    textAlign: 'center',
+    fontSize: 20,
+    fontFamily: 'Assistant',
+    width: width * (3 / 4),
+    height: height * (1 / 10),
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Signup/Signup.style.js]: Should use `padding` to set height

File: lineQueueClient/src/components/Signup/Signup.style.js [hunk 9]
```diff
@@ -1,33 +1,58 @@
...
     alignItems: 'center',
-    backgroundColor: 'white'
+    justifyContent: 'center',
   },
-  text: {
-    fontSize: 16,
-    textAlign: 'center',
-		fontFamily: 'Assistant',
-    color: '#151515'
+  inputContainerBottom: {
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Signup/Signup.style.js]: rename to `buttonContainer`

File: lineQueueClient/src/components/Signup/Signup.style.js [hunk 10]
```diff
@@ -1,33 +1,58 @@
...
 const signupStyle = StyleSheet.create({
-  containerWhite: {
-    flex: 1,
+  contentContainer: {
+    display: 'flex',
     flexDirection: 'column',
-    justifyContent: 'center',
+    justifyContent: 'space-around',
+    alignItems: 'stretch',
+    backgroundColor: 'white',
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Signup/Signup.style.js]: backgroundColor is `'white'` by default

File: lineQueueClient/src/components/Signup/Signup.style.js [hunk 11]
```diff
@@ -1,33 +1,58 @@
...
+    borderColor: '#151515',
     borderWidth: 1
   },
-  border: {
-    borderWidth: 1,
-    borderStyle: 'dashed',
-    borderColor: '#151515'
+  buttonContainer: {
+    borderWidth: 2,
+    width: width * (4 / 10),
```
Comment (shakedlokits) [inline lineQueueClient/src/components/Signup/Signup.style.js]: button `width` and `height` should be set by `padding`

File: lineQueueClient/src/styles/globals.style.js [hunk 12]
```diff
@@ -0,0 +1,18 @@
+import { StyleSheet, Dimensions } from 'react-native';
+
+const globals = StyleSheet.create({
+  sceneContainer: {
+    display:'flex',
+    width: Dimensions.get('window').width,
```
Comment (shakedlokits) [inline lineQueueClient/src/styles/globals.style.js]: To request window width and height do:
```js
let {height, width} = Dimensions.get('window')
```
and then:
```json
{
  width: width,
  height: height
}
```

File: lineQueueClient/src/styles/globals.style.js [hunk 13]
```diff
@@ -0,0 +1,18 @@
...
+    height: Dimensions.get('window').height,
+
+  },
+  text: {
+    fontSize: 16,
+    textAlign: 'center',
+		fontFamily: 'Assistant',
```
Comment (shakedlokits) [inline lineQueueClient/src/styles/globals.style.js]: bad indentation, provide uniform indentation rules

**Review by shakedlokits — CHANGES_REQUESTED** [review]
Should amend some optimization and styling issus but Good Job! 👍

File: lineQueueClient/src/components/Header/Header.js (hunk 2 above)
Comment (OphirSheriff) [reply lineQueueClient/src/components/Header/Header.js]: fixed, defined the color as main in globals.style

File: lineQueueClient/src/components/Header/Header.js (hunk 1 above)
Comment (OphirSheriff) [reply lineQueueClient/src/components/Header/Header.js]: right padding is meant to be always static

File: lineQueueClient/src/components/Header/Header.js (hunk 5 above)
Comment (OphirSheriff) [reply lineQueueClient/src/components/Header/Header.js]: fixed

File: lineQueueClient/src/components/Signup/Signup.style.js (hunk 10 above)
Comment (OphirSheriff) [reply lineQueueClient/src/components/Signup/Signup.style.js]: checked it - background isn't white by default, it's no-fill by default. backgroundColor should remain there.

File: lineQueueClient/src/components/Signup/Signup.style.js (hunk 9 above)
Comment (OphirSheriff) [reply lineQueueClient/src/components/Signup/Signup.style.js]: buttonContainer was already used for the Button Box. Renamed inputContainerButtom to buttonContainer and  what was buttonContainer to buttonBox

File: lineQueueClient/src/styles/globals.style.js (hunk 12 above)
Comment (OphirSheriff) [reply lineQueueClient/src/styles/globals.style.js]: fixed

File: lineQueueClient/src/styles/globals.style.js (hunk 13 above)
Comment (OphirSheriff) [reply lineQueueClient/src/styles/globals.style.js]: fixed

**Review by shakedlokits — APPROVED** [review]
Very good job 🥇 LGTM

**Pull request merged**